CDCgov/prime-reportstream
# 11299 - Remove inactive receivers

This PR ...

This PR removes any services from the dropdown within Daily Data and Data Dashboard that service.customerStatus !== "inactive"

Test Steps:
1. Log into RS
2. View and set an org that has receivers that are active and inactive. i.e. ca-dph
3. Click on Dashboard in the nave
4. Notice that the receiver list does not include inactive receivers

## Changes
![Screenshot 2023-09-20 at 10 18 02 AM](https://github.com/CDCgov/prime-reportstream/assets/102491809/49386aca-a9c1-4fa2-a053-ea798a18d524)

![Screenshot 2023-09-19 at 1 09 40 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/02301b21-42f0-4cf1-841b-3c75ede3a58b)

![Screenshot 2023-09-19 at 1 10 07 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/1e60da33-3190-4865-a45d-e70b785be895)


## Checklist

### Testing


## Linked Issues
- Fixes #11299
